patches/minecraft/net/minecraft/world/spawner/PatrolSpawner.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/world/spawner/PatrolSpawner.java
++++ b/net/minecraft/world/spawner/PatrolSpawner.java
+@@ -94,6 +94,7 @@
+          return false;
+       } else {
+          PatrollerEntity patrollerentity = EntityType.field_220350_aJ.func_200721_a(p_222695_1_);
++         if(net.minecraftforge.common.ForgeHooks.canEntitySpawn(patrollerentity, p_222695_1_, p_222695_2_.func_177958_n(), p_222695_2_.func_177956_o(), p_222695_2_.func_177952_p(), null, SpawnReason.PATROL) == -1) return false;
```

Comment:
Same for PatrolSpawner. `func_230253_a_` is the main method that is used to determine if a Patrol can spawn. I just hooked into `spawnPatroller` at the very end to look at the end result if the game decides it wants to spawn a Patrol or not.